meta-lmp-base/recipes-support/lmp-device-auto-register/lmp-device-auto-register/lmp-device-auto-register

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh -e
+
+TOKEN_FILE=${TOKEN_FILE-/etc/lmp-device-register-token}
+
+if [ ! -f $TOKEN_FILE ] ; then
+	echo "ERROR: Missing token file $TOKEN_FILE"
+	exit 1
+fi
+
+TOKEN=$(cat $TOKEN_FILE)
+if [ -z $TOKEN ] ; then
+	echo "ERROR: $TOKEN_FILE is empty"
+	exit 1
+fi
+
+if [ -f /var/sota/sql.db ] ; then
+	echo "ERROR: Device appears to already be registered."
+	exit 1
+fi
+
+# Done in 2 parts. This first part will remove trailing \n's and make
+# the output all space separated. The 2nd part makes it comma separated.
+[ -d /var/sota/compose-apps ] && APPS=$(ls /var/sota/compose-apps)
```

Comment:
`compose-apps` folder does not exist at the moment of bitbaking unless compose apps are preloaded. Even if they are preloaded then all apps from containers.git are preloaded, if it is so then `-a` param cam be eliminated.